pdns/dnsdist.hh

```diff
@@ -698,6 +698,8 @@ struct DownstreamState
   int tcpConnectTimeout{5};
   int tcpRecvTimeout{30};
   int tcpSendTimeout{30};
+  int checkInterval{1};
+  int lastCheck{0};
```

Comment:
Using `unsigned int` instead of `int` for these two would be nicer.

Reply:
I went for int because I am using std::stoi and there is no std::stoU... maybe I was a bit lazy.
But you are right, I will update this commit.

Reply:
We do have `pdns_stou()` in `misc.hh`, which throws `std::exception` on invalid content.